docs/README_Docker_and_RA.md

```diff
@@ -437,7 +437,7 @@ Once your Raiko instance is running, you can verify if it was started properly a
 }'
 ```
 
-Or use `./script/prove-block` like `./script/prove-block.sh taiko_a7 native 99999` to check readiness.
```

Comment:
running the native to "check for readiness" means to make sure the host & block builder without any proof, so it should not be sgx

Reply:
I am wrong to assume that in this case the response shouldn't include a proof?

This is the example response the docs provide. 

```
{
    "data": {
        "output": null,
        "proof": "0x00000206c3694ecb5c....6e0e7a36546bf98caa7bb4ac2cd4f917c2102116167e42c54849f15044c032e1c",
        "quote": "03000200000000000a000f00939a72....0a2d2d2d2d2d454e442043455254494649434154452d2d2d2d2d0a00"
    },
    "status": "ok"
}
```

If so, I can edit the PR to match the expected (no proof) output.